### PR TITLE
[UI] Migrate DatabaseSummary component to Sistient and MUI v5. Fixes #12421

### LINKED ISSUE #12421: [UI] Migrate DatabaseSummary Component to Sistient and MUI v5
**Description:** Meshery's `DatabaseSummary.js` component is using an outdated version of Material-UI (v4), which leads to increased maintenance and performance issues.

- **Expected Outcome:** Migrate the `DatabaseSummary.js` component from MUI v4 to MUI v5 and apply Sistient styling. This will simplify the code, improve maintainability, and ensure the component is responsive and accessible across devices.

- **Recommended Skills:** ReactJS, Material-UI v5, familiarity with Sistient.

- **Component Reference:** [DatabaseSummary Component](https://github.com/meshery/meshery/blob/a9d81c8099f3e9197337b90871ef0fc6a8437727/ui/components/DatabaseSummary.js#L203)

- **Mentor(s):** Lee Calcote (@leecalcote, leecalcote@gmail.com), Antonette Caldwell (@nebula-aac, pullmana8@gmail.com)

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Vaibhav701161**
***Notes for Reviewers***

This PR migrates the **DatabaseSummary.js** component to use Sistent and MUI v5.

**Changes:**
- Updated MUI v4 components to MUI v5 with Sistent equivalents (Button, Typography, useTheme).
- Wrapped the component with **UsesSistent** for consistent theming.
- No logic changes; functionality remains intact.

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Vaibhav701161** [comment]
@SAHU-01 please review this pr

**Comment by leecalcote** [comment]
Thanks, @Vaibhav701161.

@sudhanshutech are we still doing this?

```
Wrapped the component with UsesSistent for consistent theming.
```

**Comment by leecalcote** [comment]
@sudhanshutech are we still doing this?

> Wrapped the component with UsesSistent for consistent theming.

**Comment by leecalcote** [comment]
> @sudhanshutech are we still doing this?
> 
> > Wrapped the component with UsesSistent for consistent theming.

@sudhanshutech, asking again: are we still doing this?

File: ui/components/DatabaseSummary.js [hunk 1]
```diff
@@ -121,62 +102,30 @@ const DatabaseSummary = (props) => {
...
-  });
-
-  const customInlineStyle = {
-    marginBottom: '0.5rem',
-    marginTop: '1rem',
-  };
-
   return (
-    <>
-      <div className={StyleClass.toolWrapper} style={customInlineStyle}>
```
Comment (sudhanshutech) [inline ui/components/DatabaseSummary.js:152]: its good that you have removed the class uses but have you ensure that are converted to style components and there styles are not lost?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
> > @sudhanshutech are we still doing this?
> > > Wrapped the component with UsesSistent for consistent theming.
> 
> @sudhanshutech, asking again: are we still doing this?

yes until the all pages are completed and then we will remove from everywhere and use it in single place that is app.js.

**Comment by sudhanshutech** [comment]
@Vaibhav701161 checking in again...

**Pull request closed without merging**